lib/middleware/api-session.ts

```diff
@@ -12,28 +12,6 @@ import Interface = require('../interface');
 var SESSIONSTORE = new SessionStore(conf.get('expiration'));
 
 // PUBLIC FUNCTIONS
-export function createSession(req: Interface.IOurRequest,
```

Comment:
When you have different routes for subscribe and unsubscribe, you could have different functions called for each. However, we now have just one route for `POST /realtime`, so we have just one function called by that route. If there's a way in restify to tell the difference earlier, I'll be happy to revert to the old setup, I just didn't think there was.